specs/DesignDiscussions/InputInlineEditability.md

```diff
@@ -0,0 +1,159 @@
+In both cases we might have duplicate readable fields and input fields.
+
+We want to provide better experience to the user where he can read the fields and can also seamlessly update the values at the same place. 
+
+UI of showing data and taking user input will become simplistic. 
+
+ 
+## Current experience:
+
+As shown in the below picture, An adaptive card displays data for Customer name, Est. Revenue and Est. Clode Date. If user wants to update those information, he will have to click on 'Edit' button which will open some form and there he can update values for 'Est. Revenue' etc. and Click on Save to send data back to the bot service.
+
+![img](./assets/InputInlineEdit/InputInlineEditScenario1.PNG)
+
+
+As of now, Adaptive cards support `Input` fields to collect user input. This is how current user experience of `Input` fields looks like. 
+
+ ![img](./assets/InputInlineEdit/InputInlineEditScenario2.PNG)
+
+## Proposed experience:
+
+We will allow bot developer to enhance the user experience of all `Input` fields in Adaptive card (such as `Input.Text`, `Input.Number`, `Input.Date`, `Input.Time`, `Input.Toggle` and `Input.Choiceset`) in way that, these input fields can appear just as readable fields when user is not taking any action and when user clicks or focusses or clicks on them, it allows user to update those fields and then user can use action buttons like Action.Submit/Action.Execute present on the card to send data back to the bot.
+
+This is the flow for user to interact with inline editable fields:-
+1. This is an adaptive showing information like "Contact", "Email", "Est. Closing date" etc. with inline Editable style where input fields are displayed as **readable fields** in the **default** state.
+
+![img](./assets/InputInlineEdit/InputInlineEditScenario3.PNG)
+
+2. When user hovers on the card, 
+	* we will show outer outline on that card.
+	* we will show a lighter color outline to all the input fields which are inline editable.
+	
+	This will guide user to the input fields which he can go an update right on the card.
+	
+![img](./assets/InputInlineEdit/InputInlineEditScenario4.PNG)
+
+3. When user takes hovers over the input field, we will show a darker outline in that input field to guide user that the field is activated and he can now edit the field.
+
+![img](./assets/InputInlineEdit/InputInlineEditScenario5.PNG)
+
+4. User can go and edit the field.
+
+![img](./assets/InputInlineEdit/InputInlineEditScenario6.PNG)
+
+5. When user has updated the input field, Action.Submit or Action.Execute button such as "Save" which has  `disabledUnlessAssociatedInputsChange` property (https://github.com/microsoft/AdaptiveCards/issues/7103) will get enabled automatically. It will guide user to hit the button to commit the changes made on the card.
```

Comment:
```suggestion
5. When the user has updated the input field, Action.Submit or Action.Execute button such as "Save" which has `disabledUnlessAssociatedInputsChange` property (https://github.com/microsoft/AdaptiveCards/issues/7103) will get enabled automatically. It will guide user to hit the button to commit the changes made on the card.
```